Client/Configuration/index.ts

```diff
@@ -28,6 +28,9 @@ export class Configuration {
 	async getPortalFeatures(): Promise<model.PaxpayFeature[] | model.ErrorResponse> {
 		return await this.connection.get<model.PaxpayFeature[]>(`${this.folder}/portal`)
 	}
+	async getConfigValueFromKey(category: string, key: string): Promise<any | model.ErrorResponse> {
```

Comment:
would it be worth constraining the types here?

Reply:
https://github.com/pax2pay/mpay-server/blob/master/internal-api/src/main/java/com/xmltravel/multicommerce/config/config/access/ConfigKey.java

its not documented yet so eh, ok